src/lib/Form/Type/Content/ContentEditType.php

```diff
@@ -43,26 +43,32 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('publish', SubmitType::class, ['label' => 'Publish']);
 
-        if ($options['drafts_enabled']) {
-            $builder
-                ->add('saveDraft', SubmitType::class, [
-                    'label' => /** @Desc("Save draft") */ 'save_draft',
-                    'attr' => ['formnovalidate' => 'formnovalidate'],
-                ])
-                ->add('cancel', SubmitType::class, [
-                    'label' => /** @Desc("Cancel") */ 'cancel',
-                    'attr' => ['formnovalidate' => 'formnovalidate'],
-                ])
-                ->add('autosave', SubmitType::class, [
-                    'label' => /** @Desc("Autosave") */ 'autosave',
-                    'attr' => [
-                        'hidden' => true,
-                        'formnovalidate' => 'formnovalidate',
-                    ],
-                    'translation_domain' => 'content_edit',
-                ]);
-            $builder->addEventSubscriber(new SuppressValidationSubscriber());
+        if (!$options['drafts_enabled']) {
+            return;
         }
+
+        $builder
+            ->add('saveDraft', SubmitType::class, [
+                'label' => /** @Desc("Save draft") */ 'save_draft',
+                'attr' => ['formnovalidate' => 'formnovalidate'],
+            ])
+            ->add('cancel', SubmitType::class, [
+                'label' => /** @Desc("Cancel") */ 'cancel',
+                'attr' => ['formnovalidate' => 'formnovalidate'],
+            ]);
+
+        if (!($options['autosave_disabled'] ?? false)) {
```

Comment:
is that check needed if there is default value in `configureOptions`. Is `if (!($options['autosave_disabled'])`, not enough?

I also now wonders, if it shouldnt be `autosave_enabled`, to be in line with `drafts_enabled`, but no strong feelings on it.

Reply:
>I also now wonders, if it shouldnt be autosave_enabled, to be in line with drafts_enabled, but no strong feelings on it.
Fine by me. Changed.